lms/djangoapps/courseware/permissions.py

```diff
@@ -3,7 +3,10 @@
 """
 
 from bridgekeeper import perms
-from .rules import HasAccessRule
+from .rules import HasAccessRule, HasStaffAccessToContent
 
 VIEW_COURSE_HOME = 'courseware.view_course_home'
+MASQUERADE_AS_STUDENT = 'courseware.masquerade_as_student'
+
 perms[VIEW_COURSE_HOME] = HasAccessRule('load')
+perms[] = HasStaffAccessToContent()
```

Comment:
@cpennington this is missing MASQUERADE_AS_STUDENT

also, could we perhaps call this something like MASQUERADE_AS_STUDENT_WITH_LABORATORY?
per your suggestion, I am just using the regular HasAccessRule for some other staff access checks, but i have another can_masquerade permission that would need to be somehow differentiated from this one, if we didn't want to add more perms to this yet

Reply:
I think let's just use one permission `MASQUERADE_AS_STUDENT`, and have all the places that call that permission trigger the laboratory check.